### PR TITLE
Remove hardcoded eu region API url

### PR DESCRIPTION
**Pull request by jpserra**
https://onfido.atlassian.net/browse/BIOMANU-1341

# Problem

Auth component is using hardcoded API URL

# Solution

The correct url to call can be fetched from the sdkToken used to initialize
the OnfidoAuth sdk.

This commit removes the EU BaseURL that was being used allowing the auth
sdk to be used in any given region.
## Checklist

_put `n/a` if item is not relevant to PR changes_

- [x] Has the CHANGELOG been updated?
- [ ] Has the README been updated?
- [ ] Has the CONTRIBUTING doc been updated?
- [ ] Has the RELEASE_GUIDELINES been updated?
- [ ] Has the TESTING_GUIDELINES been updated?
- [ ] Has the MIGRATION doc been updated for any MAJOR breaking changes?
- [ ] Has the MIGRATION doc been updated for any MINOR breaking changes, including any translation strings or keys changes?
- [ ] Have any new automated tests been implemented or the existing ones changed?
- [ ] Have any new manual tests been written down or the existing ones changed?
- [ ] Have any new strings been translated or the existing ones changed?

### CODE REVIEW

File: src/components/Auth/Auth.tsx [hunk 1]
```diff
@@ -25,6 +25,7 @@ const AuthCapture: FunctionComponent<Props> = (props) => {
   const [, { findStep }] = useSdkOptions()
   const { translate } = useLocales()
   const retries = findStep('auth')?.options?.retries || 3
+  const apiUrl = getUrlsFromJWT(props.token).onfido_api_url
```
Comment (stefaniacardenas) [inline src/components/Auth/Auth.tsx:28]: The URLs should be already available here from the Redux store, are they not?

File: src/components/Auth/AuthConfig.tsx [hunk 2]
```diff
@@ -1,11 +1,11 @@
 import { loader } from './assets/loader'
 import { success } from './assets/success'
 import { borderRadius, color } from '@onfido/castor'
+import type { ApiRawError, SuccessCallback } from '~types/api'
 import { FaceTecCustomization } from '~auth-sdk/FaceTecSDK.js/FaceTecCustomization'
 import { FaceTecSDK } from '~auth-sdk/FaceTecSDK.js/FaceTecSDK'
 import { UICustomizationOptions } from '../../types/ui-customisation-options'
 
-export const BaseURL = process.env.AUTH_URL
```
Comment (stefaniacardenas) [inline src/components/Auth/AuthConfig.tsx:8]: `AUTH_URL` should be remove from `webpack.config.babel.js`

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]
This should point at `development`

File: src/components/Auth/Auth.tsx (hunk 1 above)
Comment (jpserra) [reply src/components/Auth/Auth.tsx]: right, makes sense. pushed another commit let me know if that makes more sense

**Review by stefaniacardenas — CHANGES_REQUESTED** [review]
Looks good to me, it just needs a changelog entry :)

**Review by stefaniacardenas — APPROVED** [review]

**Pull request merged**